test/integration/utils/taskruns.go

```diff
@@ -68,14 +72,19 @@ func (t *TestBuild) GetTRTillDesiredReason(buildRunName string, reason string) e
 		pollTRTillCompletion = func() (bool, error) {
 
 			trReason, err := t.GetTRReason(buildRunName)
+			// Not found means that we may not have a TaskRun for this BuildRun yet
+			if kerrors.IsNotFound(err) {
+				fmt.Printf("could not find TaskRun for BuildRun %s", buildRunName)
+				return false, nil
+			}
 			if err != nil {
 				return false, err
 			}
 
 			if trReason == reason {
 				return true, nil
 			}
-
+			fmt.Printf("expecting TaskRun reason %s for BuildRun %s, got %s", reason, buildRunName, reason)
```

Comment:
this is a good point, but also it pollutes the whole integration test output. Can you please merge the following PR https://github.com/shipwright-io/build/pull/484, that would fulfill what you are looking for.